site/content/overview/product-telemetry.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Product Telemetry"
+description: "Learn how NGINX Gateway Fabric collects product telemetry to identify areas to focus development on improving."
+weight: 500
+toc: true
+---
+
+## Overview
+
+NGINX Gateway Fabric collects telemetry by default, which allows its developers to understand what features are most popular with its user base. This data is used to triage development work, prioritizing features and functionality that will positively impact the most people.
+
+Telemetry data is collected once every 24 hours and sent to a service managed by F5. Personally identifiable information (PII) is **not** collected. The list of data points that are collected can be seen below.
+
+**If you would prefer to not have data collected, you can [opt-out](#opt-out) when installing NGINX Gateway Fabric.**
+
+## Collected data
+
```

Comment:
Do we care about also putting `NGINX Gateway Fabric Version` as one of the collected data points?